src/elements/pages/dev/namespace/rvt-namespace-settings.ts

```diff
@@ -39,6 +39,7 @@ const PORT_PROTOCOLS: DropDownSelection<cloud.ProxyProtocol>[] = [
 ];
 
 const LOBBY_COUNT_MAX = 32768 - 1;
+const MAX_MAX_PLAYERS_PER_IP = 2 ** 32 - 1; // u32
```

Comment:
Any reason to not use `u32::MAX` for clarity?
```suggestion
const MAX_MAX_PLAYERS_PER_IP = u32::MAX;
```

Reply:
This is typescript lol